integration/network/service_test.go

```diff
@@ -159,3 +149,17 @@ func swarmIngressReady(client client.NetworkAPIClient) func(log poll.LogT) poll.
 		return poll.Success()
 	}
 }
+
+func noServices(client client.ServiceAPIClient) func(log poll.LogT) poll.Result {
```

Comment:
Erm, this function doesn't seem to be called?

Reply:
I kept the new API so that TestServiceWithIngressNetwork will call it. Let me update it.